build/dependencies.props

```diff
@@ -6,80 +6,80 @@
     <AngleSharpPackageVersion>0.9.9</AngleSharpPackageVersion>
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
     <InternalAspNetCoreSdkPackageVersion>2.1.0-preview3-17032</InternalAspNetCoreSdkPackageVersion>
-    <MicrosoftAspNetCoreAntiforgeryPackageVersion>2.1.0-preview3-32233</MicrosoftAspNetCoreAntiforgeryPackageVersion>
```

Comment:
Why is this file changing?

Reply:
It's the only deterministic way to tell if your repo isn't already broken or broken because one of the dependencies changed.